<launch>
    <arg name="dischargeRateArg" default="10" />
    <arg name="initialChargeArg" default="100" />
    <arg name="chargeRateArg"    default="10"  />
    <node pkg="robots" type="robot.py" name = "$(anon robot)" output="screen" >
        <param name="discharge_rate" value="$(arg dischargeRateArg)" />
        <param name="initial_battery_level" value="$(arg initialChargeArg)" />
        <param name="charge_rate" value="$(arg chargeRateArg)" />
    </node>
</launch>